### ISSUE TITLE
values for buienalarm seem 100x too high

### ISSUE THREAD

**Issue by Mrten** [aex351/home-assistant-neerslag-app#21]
had the neerslag card, upgraded to neerslag app but both seem to have the buienalarm values 100 time too high, see image from nov 20 (no 60mm storm)

![9770A1BF-2904-4DBC-8A16-5EAC28EE041E](https://user-images.githubusercontent.com/1662703/142738759-c3d02689-b81b-4b00-876f-bf49b6f54f6f.jpeg)

```
  - platform: command_line
    command: python3 -c "import requests; import json; import random; dataRequest = requests.get('https://cdn-secure.buienalarm.nl/api/3.4/forecast.php?lat=XXXXX&lon=XXX&region=nl&c='+str(random.randint(0,9999))).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":'+dataRequest+'}'; print(data);"
    name: Neerslag_Buienalarm_Regen_Data
    json_attributes:
      - data
    value_template: 'last_changed: {{states.sensor.neerslag_buienalarm_regen_data.last_changed | default(now())}}'
    scan_interval: 300
  - platform: command_line
    command: python3 -c "import requests; import json; import random; dataRequest = requests.get('https://gpsgadget.buienradar.nl/data/raintext?lat=XXX&lon=XXX&c='+str(random.randint(0,9999))).text; dataRequest = dataRequest.replace('\r\n',' '); data = '{\"data\":\"'+dataRequest+'\"}'; print(data);"
    name: Neerslag_Buienradar_Regen_Data
    json_attributes:
      - data
    value_template: 'last_changed: {{states.sensor.neerslag_buienradar_regen_data.last_changed | default(now())}}'
    scan_interval: 300
```

(location edited to protect the innocent)

**Comment by aex351** [comment]
To start. Please remove the `Neerslag Card` configuration from `configuration.yaml` if you are using the `Neerslag App`.  The `Neerslag App` doesn't require the setup of custom sensors.

To continue. I am not able to reproduce the issue you describe. The `Neerslag App` fetches the information from Buienalarm and/or Buienradar as-is based on your location settings. If you are experiencing incorrect values then this probably is caused by either 1) incorrect location settings in Home Assistant or 2) Buienalarm giving invalid data based on your location settings.

Both the `Neerslag App` and the `Neerslag Card` are currently not modifying the data from both Buienalarm and Buienradar.

**Comment by Mrten** [comment]
Thanks for the suggestion, removed the config from configuration.yaml.  The 100x too high problem was the reason I went from Card to App.

Then, the Neerslag App (the card in lovelace) only started working when I removed the _2 suffix from the entities, both in the configuration > entities as in the lovelace card config.

(For the neerslag Card I had created a sensor.neerslag_buienalarm_regen_data entity as seen above, so my guess is that the Neerslag App created sensor.neerslag_buienalarm_regen_data_2. So I put the _2 entities in the lovelace card config but they did not work).

I thought it curious, but I'll leave it up to you if you want to ignore that and close this bug.
